packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -10,7 +10,11 @@
 
 'use strict';
 
+const ReactFeatureFlags = require('shared/ReactFeatureFlags');
```

Comment:
@acdlite Did you say we need to put these inline now? I don't remember why.

Reply:
I copy-pasted this from elsewhere so if we do, we should codemod the rest of them.